projects/cypress/CY8CKIT_064S0S2_4343W/mtb/aws_demos/afr.mk

```diff
@@ -180,7 +180,8 @@ SOURCES+=\
 	$(wildcard $(CY_AFR_ROOT)/demos/mqtt/*.c)\
 	$(wildcard $(CY_AFR_ROOT)/demos/network_manager/*.c)\
 	$(wildcard $(CY_AFR_ROOT)/demos/tcp/*.c)\
-	$(wildcard $(CY_AFR_ROOT)/demos/shadow/*.c)
+	$(wildcard $(CY_AFR_ROOT)/demos/shadow/*.c)\
+	$(wildcard $(CY_AFR_ROOT)/demos/coreMQTT/*.c)
```

Comment:
When you rebase you won't need this. The rebase has retry utils and transport secure sockets also added to Cypress.
You can just add `$(AMAZON_FREERTOS_PATH)demos/coreMQTT/mqtt_demo_mutual_auth.c \` right underneath the mqtt_demo_plaintext.c in aws_demos.mk.

Reply:
Added